### PR TITLE
Add single-output-sway helper script

### PR DESCRIPTION
**Pull request by lack**
I have read and understood CONTRIBUTING.md and its associated documents.

### CODE REVIEW

File: examples/single-output-sway [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+fi
+
+OUTPUTS_TO_RECONNECT=()
+restore_outputs() {
+    echo "Arraylen : ${#OUTPUTS_TO_RECONNECT[@]}"
+    [[ ${#OUTPUTS_TO_RECONNECT[@]} -ge 1 ]] || return
+    echo "Restoring original output state"
+    for output in "${OUTPUTS_TO_RECONNECT[@]}"; do
+        echo "Re-enabling output $output"
+        #$SWAYMSG output "$output" enable
```
Comment (any1) [inline examples/single-output-sway:19]: Is this meant to be a comment?

File: examples/single-output-sway [hunk 2]
```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+WAYVNCCTL=${WAYVNCCTL:-wayvncctl}
+SWAYMSG=${SWAYMSG:-swaymsg}
+
+SWAY_HAS_UNPLUG=
```
Comment (any1) [inline examples/single-output-sway:6]: Using `true`/`false` for this would perhaps look a bit nicer.

**Review by any1 — APPROVED** [review]
LGTM

**Comment by lack** [comment]
Thanks!

I will try to test it on sway 1.8 first before merging.

**Comment by lack** [comment]
Don't merge yet; more testing needed

**Pull request merged**